### DISCUSSION TITLE
A list of potential applications

### DISCUSSION THREAD

**Discussion by sidcode** [mprimi/portable-secret#1] [Ideas]
This thread intends to curate all the ways we can share our portable secrets among each other.

**Comment by ripgith** [comment]
I frequently need to send files (documents, spreadsheets) that contain PII to other parties that are not on a trusted network. This looks like a good alternative to using an external tool to encrypt/decrypt or an intermediary such as dropbox. I'll give it a try the next chance I have.

I also wonder about using this with a browser-based note-taking app so I can have totally private well-structured notes available whereever I am and whatever device I'm using. Might need to be in combination with some external shared folder or perhaps an auto-synching app like Syncthing.

**Reply by dumblob** [reply]
As a side note - I have proposed "rich text" as a directly in-browser-renderable content "type" in https://github.com/mprimi/portable-secret/issues/17#issue-1509103374 .

**Comment by ripgith** [comment]
I tend to get carried away (and should be...)

A far-fetched idea is to actually store something like a sqlite DB, perhaps with a even a UI to query it.
Obviously this is not performant. And is not something that would allow "your grandmother" to update a field and send it back to you.

**Comment by ripgith** [comment]
Add some type of automated versioning to the contents, along with diffs, reverts, etc. Going too? far, even a blockchain for auditing/verification.
